### ISSUE TITLE
Cask install path note

### ISSUE THREAD

**Issue by christianboyle** [mariuszostrowski/sketchexporter#1]
Great tool! Ran into a bit of trouble installing this due to some (possibly) common issues:
1. I hadn't updated sketchtool in a while and was on version 1.2 (current is 1.4). For some reason I had alias sketchtool="sketchtool/bin/sketchtool" (which took a while to remember, sigh). I can't recall why this alias was necessary, but others might have it too, so be sure to remove that prior to upgrading sketchtool.
2. If you are using the brew cask install method (which I also forgot I used, lol) you will need to change the path in [document.wflow, line 1118](https://github.com/mariuszostrowski/sketchexporter/blob/master/Sketch%20Exporter.workflow/Contents/document.wflow#L1118): 

```
<string>/opt/homebrew-cask/Caskroom/sketch-tool/latest/sketchtool/bin/sketchtool export ${2} --output="${3}" "${1}"
/usr/bin/osascript -e 'display notification "All done!" with title "Sketch Exporter"'</string>
```

To summarize the fix:
1. remove the old alias
2. remove the old sketchtool install
3. fresh sketchtool install with `brew cask install sketch-tool`
4. install sketchexporter in ~/Library/System
5. alter line 1118 to point to the cask install path

Thanks again for the handy tool, much better than opening the sketchtool readme every time I want to export a few artboards.
